Scripts/RemediationScripts/Remediate-SetClassicFrontDoorMinTLSVersion.ps1

```diff
@@ -416,6 +418,7 @@ function Set-FrontDoorRequiredTLSVersion
                                                     @{N='ResourceGroupName';E={$item.Id.Split('/')[4]}},
                                                     @{N='ResourceName';E={$item.Id.Split('/')[8]}},
                                                     @{N='HostName';E={$item.Name}},
+                                                    @{N='CustomHttpsProvisioningState';E={$item.CustomHttpsProvisioningState}},
```

Comment:
why this is needed?

Reply:
Hi Saurabh, so basically, this prop is needed to check if custom http configuration has been enabled. because if its enabled then only we can check what is the TLS version on it. 
Control gets fail in both of the case, if TLS version is not correct OR https configuration has not been enabled. 
because until its enabled TLS encryption will not take place